index.d.ts

```diff
@@ -626,3 +626,49 @@ interface KVNamespace {
     cursor: string;
   }>;
 }
+
+interface DurableObjectEntries<T> {
+  [key: string]: T;
+}
+
+interface DurableObjectListOptions {
+  start: string;
+  end: string;
+  reverse: boolean;
+  limit: number;
+}
+
+interface DurableObjectStorage {
+  get<ExpectedValue = unknown>(key: string): Promise<ExpectedValue>;
```

Comment:
I know the KV interface did this for whatever reason, but I don't think naming a type parameter ExpectedValue or GivenValue adds much information versus just Value or even V